content/docs/command-reference/add.md

```diff
@@ -33,7 +35,8 @@ option to avoid this, and `dvc commit` to finish the process when needed).
 > intermediate and final results (like ML models).
 
 After checking that each `target` hasn't been added before (or tracked with
-other DVC commands), a few actions are taken under the hood:
+other DVC commands), a few actions are taken under the hood (if `--to-remote` is
+not provided):
```

Comment:
I think let's revert this change and be a bit more specific by mentioning to-remote in steps 2 and 3 instead.